test/wiki-feed-parser.test.js

```diff
@@ -0,0 +1,146 @@
+const parser = require('../src/backend/utils/wiki-feed-parser');
+global.fetch = require('node-fetch');
+
+const mockFeed = `################# Failing Feeds Commented Out [Start] #################
+
+#Feed excluded due to getaddrinfo ENOTFOUND s-aleinikov.blog.ca s-aleinikov.blog.ca:80
+#[http://s-aleinikov.blog.ca/feed/atom/posts/]
+#name=Sergey Aleinikov
+
+
+#Feed excluded due to getaddrinfo ENOTFOUND ejtorre.blog.ca ejtorre.blog.ca:80
+#[http://ejtorre.blog.ca/feed/rss2/posts/]
+#name=Eugene Torre
+
+
+#Feed excluded due to getaddrinfo ENOTFOUND rickeyre.ca rickeyre.ca:80
+#[http://rickeyre.ca/open-source-feed.xml]
+#name=Rick Eyre
+
+[http://kopay.wordpress.com/category/sbr600-win2011/feed]
+name=Pirathapan Sivalingam
+
+
+[http://jessefulton.wordpress.com/category/SBR600/feed/]
+name=Jesse Fulton
+
+
+[http://eric-spo600.blogspot.com/feeds/posts/default]
```

Comment:
I pass in a feed list similar with this format.  From what I understand the getData() is supposed to parse this information and as you said, return an array of strings.  However, it doesn't do this and just returns an array with one string where all the newline characters are escaped.